packages/vm/src/runTx.ts

```diff
@@ -144,45 +128,31 @@ export async function runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
     const castedTx = <AccessListEIP2930Transaction>opts.tx
 
     for (const accessListItem of castedTx.AccessListJSON) {
-      const address = toBytes(accessListItem.address)
-      state.addWarmedAddress(address)
+      this.evm.evmJournal.addPreWarmedAddress(accessListItem.address, true)
       for (const storageKey of accessListItem.storageKeys) {
-        state.addWarmedStorage(address, toBytes(storageKey))
+        this.evm.evmJournal.addPreWarmedSlot(accessListItem.address, storageKey, true)
```

Comment:
Is it intentional that we're refactoring `addWarmedStorage` to `addPreWarmedSlot` (since we have a `addWarmedStorage` method on the evmJournal)?